lib/veritech-server/src/server.rs

```diff
@@ -288,6 +289,7 @@ impl Server {
         async_nats::jetstream::consumer::pull::Config {
             durable_name: Some(CONSUMER_NAME.to_owned()),
             filter_subject: subject::incoming(subject_prefix).to_string(),
+            max_deliver: CONSUMER_MAX_DELIVERY,
```

Comment:
@fnichol I messed around with including `backoff` here, but it causes it to retry if the first timer expires before we ack/nack, causing quintuplicate function runs. If I understand correctly, we aren't acking until after execution, so we would be racing the timeout.

Reply:
This sounds right to me. I think right sizing the delivery max is a good plan B.

Reply:
Ah, I think the reason for that first timer expiring comes down to the `act_wait` value, which is 30 seconds by default. We fire `AckKind::Progress` messages by default every (30 -1) seconds but only *after* we start processing the message. So that time to first `AckKind::Progress` message is `time_after_pull_waiting_for_instance + 30` seconds.